leasing/models/land_use_agreement.py

```diff
@@ -0,0 +1,447 @@
+        related_name="+",
+        null=True,
+        blank=True,
+        on_delete=models.PROTECT,
+    )
+
+    # In Finnish: Päätöspäivämäärä
+    decision_date = models.DateField(
+        verbose_name=_("Decision date"), null=True, blank=True
+    )
+
+    # In Finnish: Pykälä
+    section = models.CharField(
+        verbose_name=_("Section"), null=True, blank=True, max_length=255
+    )
+
+    class Meta:
+        verbose_name = pgettext_lazy("Model name", "Land use agreement decision")
+        verbose_name_plural = pgettext_lazy(
+            "Model name", "Land use agreement decisions"
+        )
+
+
+class LandUseAgreementAddress(AbstractAddress):
+    land_use_agreement = models.ForeignKey(
+        LandUseAgreement,
+        verbose_name=_("Land use agreement"),
+        related_name="addresses",
+        on_delete=models.CASCADE,
+    )
+
+    # In Finnish: Ensisijainen osoite
+    is_primary = models.BooleanField(verbose_name=_("Is primary?"), default=False)
+
+    class Meta:
+        verbose_name = pgettext_lazy("Model name", "Land use agreement address")
+        verbose_name_plural = pgettext_lazy(
+            "Model name", "Land use agreement addresses"
+        )
+
+
+class LandUseAgreementConditionType(NameModel):
+    """
+    In Finnish: Hallintamuoto
+    """
+
+
+class LandUseAgreementCondition(TimeStampedSafeDeleteModel):
+    """
+    In Finnish: Lasku
+    """
+
+    land_use_agreement = models.ForeignKey(
+        LandUseAgreement,
+        verbose_name=_("Land use agreement"),
+        related_name="conditions",
+        on_delete=models.PROTECT,
+    )
+
+    # In Finnish: Hallintamuoto
+    type = models.ForeignKey(
+        LandUseAgreementConditionType,
+        verbose_name=_("Form of management"),
+        related_name="+",
+        null=True,
+        blank=True,
+        on_delete=models.PROTECT,
+    )
+
+    # In Finnish: Valvontapvm
+    due_date = models.DateField(verbose_name=_("Due date"), null=True, blank=True)
+
+    # In Finnish: Valvottu pvm
+    sent_date = models.DateField(verbose_name=_("Sent date"), null=True, blank=True)
+
+    # In Finnish: Kommentti
+    description = models.TextField(verbose_name=_("Description"), null=True, blank=True)
+
+
+class LandUseAgreementLitigant(TimeStampedSafeDeleteModel):
+    land_use_agreement = models.ForeignKey(
+        LandUseAgreement,
+        verbose_name=_("Land use agreement"),
+        related_name="litigants",
+        on_delete=models.CASCADE,
+    )
+
+    # In Finnish: Jaettava / Osoittaja
+    share_numerator = models.PositiveIntegerField(verbose_name=_("Numerator"))
+
+    # In Finnish: Jakaja / Nimittäjä
+    share_denominator = models.PositiveIntegerField(verbose_name=_("Denominator"))
+
+    # In Finnish: Viite
+    reference = models.CharField(
+        verbose_name=_("Section"), null=True, blank=True, max_length=255
+    )
+
+    class Meta:
+        verbose_name = pgettext_lazy("Model name", "Land use agreement lithigant")
```

Comment:
`lithigant` -> `litigant`, also two rows down

Reply:
Lots of lithigants :D